src/doc/gui_manual/Preferences/File_Locations.rst

```diff
@@ -0,0 +1,314 @@
+  hits the **Export Host** button from the **Host Profiles** window.
+* Read: On VisIt_ startup. All host profiles in ``VUSER_HOME/hosts/host*.xml`` are read
+  on VisIt_ startup but this can be overridden by ``-noconfig``. Users should be
+  aware of this behavior. If the user passes ``-noconfig`` for the purposes of
+  avoiding the loading of preferences, s/he will also be without any host profiles.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_
+
+.. _file_locations_visitrc:
+
+VisIt :ref:`Run Commands (rc) <visitrc_file>` File
+""""""""""""""""""""""""""""""""""""""""""""""""""
+* Location and file name: ``VUSER_HOME/visitrc``
+* Purpose: Holds Python code to be executed each time VisIt_ is launched.
+* Written: Whenever user hits the **Update Macros** button in the
+  :ref:`Command Window <Command_Line_Window>`.
+* Read: On VisIt_ startup of the CLI.
+* Format: Python source code. However, there is no ``.py`` file extension in the
+  file name.
+
+:ref:`Command Window <Command_Line_Window>` Tabs Script Files
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/script<K>.py`` where ``K`` is an
+  integer in the range [1...8].
+* Purpose: Hold the python code associated with each tab in the
+  **Command Window**.
+* Written: When user :ref:`saves settings <How to Save Settings>`.
+* Read: On VisIt_ startup but this can be overridden by ``-noconfig``.
+* Format: Python source code.
+
+:ref:`Color Table <Color_tables>` Files
+"""""""""""""""""""""""""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/<color-table-name>.ct``
+* Purpose: Store a single color table for easy sharing with other users.
+* Written when the user hits the **Export** button in the
+  :ref:`color table window <fig-MakingItPretty-ColorTables>` from
+  :menuselection:`Controls -> Color table...`.
+* Read: On VisIt_ startup. All color table files in ``VUSER_HOME/*.ct``
+  are read and loaded into VisIt_. However, this behavior is overridden
+  by ``-noconfig``.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_ specifying the
+  :ref:`colors and color control points <Color_tables>` for the color table.
+
+`Custom Plugin <http://visitusers.org/index.php?title=Building_plugins_using_CMake>`_ Files
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+* Location and file name(s): There are separate directories in ``VUSER_HOME``
+  for *private*, user-specific operator, database and plot plugins. These are
+
+  * ``VUSER_HOME/<visit-version>/<visit-arch>/plugins/operators/``
+  * ``VUSER_HOME/<visit-version>/<visit-arch>/plugins/databases/``
+  * ``VUSER_HOME/<visit-version>/<visit-arch>/plugins/plots/``
+
+  where ``<visit-version>`` and  ``<visit-arch>`` are the VisIt_ version number
+  and VisIt_ architecture moniker except on Windows where those sub-directories
+  are not present.
```

Comment:
Windows doesn't use the plugins subdir either. It is just
VUSER_HOME/operators
VUSER_HOME/plots
VUSER_HOME/databases